src/deploy/function_guest_create_multiple.sql

```diff
@@ -0,0 +1,34 @@
+BEGIN;
+
+CREATE FUNCTION maevsi.guest_create_multiple(
```

Comment:
Good file name for our file structure, the function name should be close to the mutation names Postgraphile autogenerates though (see `create_guest`)
```suggestion
CREATE FUNCTION maevsi.create_guests(
```